CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.8.0
+
+- Update `Charge.refund/3` type so that it matches the function behavior
```

Comment:
beware of the keyword `behaviour` though (elixir behaviour, the one that always spelling in British)

Reply:
Thanks!